tests/system/action/meeting/test_update.py

```diff
@@ -443,6 +467,22 @@ def test_update_group_d_no_permissions(self) -> None:
         self.assert_status_code(response, 403)
         assert "Missing permission:" in response.json["message"]
 
+    def test_update_group_d_orga_permissions(self) -> None:
```

Comment:
Why not use a base permission test here too?

Reply:
Group d creates an error that the base permission test doesn't recognize.
Instead of rewriting it I opted to use the same method that was used for the other group d tests already.